lib/message/decrypt.js

```diff
@@ -93,7 +93,7 @@ export async function decryptMessageLegacy(options) {
 export async function decryptMIMEMessage(options) {
     const { data: rawData, verified: embeddedVerified, errors } = await decryptMessageLegacy(options);
 
-    const { body, mimetype, verified: pgpVerified, attachments, encryptedSubject } = await processMIME(
+    const { body, mimetype, verified: pgpVerified, attachments, encryptedSubject, signature } = await processMIME(
```

Comment:
Since `decryptMessageLegacy` also return a signatures array, you should probably then concatenate the two arrays together, in this function. This corresponds to the fact that a PGP signature can appear both in the MIME structure and in the PGP message (cf. respectively `pgpVerified` and `embeddedVerified`, confusingly enough).
As written, you're only returning `signature` if it's in the MIME structure, which is the less common of the two.